pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -538,7 +538,7 @@ public static class Minion {
      * Service token for accessing protected controller APIs.
      * E.g. null (auth disabled), "Basic abcdef..." (basic auth), "Bearer 123def..." (oauth2)
      */
-    public static final String CONFIG_OF_TASK_AUTH_TOKEN = "task.auth.token";
+    public static final String CONFIG_TASK_AUTH_NAMESPACE = "task.auth";
```

Comment:
use `task.auth.namespace`?

Reply:
unfortunately, can't due to compatibility needs with `task.auth.token` which assume that there will a `.token` available after whichever namespace/prefix